lib/vintage_net_wizard/backend/default.ex

```diff
@@ -59,10 +59,11 @@ defmodule VintageNetWizard.Backend.Default do
     config = VintageNet.get_configuration("wlan0")
 
     with {:ok, wifi} <- Map.fetch(config, :wifi),
-         {:ok, _} <- Map.fetch(wifi, :ssid) do
+         {:ok, _} <- Map.fetch(wifi, :ssid),
+         {:ok, mode} when mode !== :host <- Map.fetch(wifi, :mode) do
```

Comment:
If seems like this could also be written:

```
config = VintageNet.get_configuration("wlan0")

get_in(config, [:wifi, :ssid]) != nil and get_in(config, [:wifi, :host]) != :host
```

Reply:
Done